src/odemis/acq/fastem.py

```diff
@@ -147,12 +147,24 @@ def _calculate_field_indices(self):
         field_size = (field_res[0] * px_size[0],
                       field_res[1] * px_size[1])
 
-        # Note: floating point errors here, can result in an additional row or column of fields (that's fine)
+        # Note: Megafields get asymmetrically extended towards the right and bottom.
```

Comment:
For info, the code of tiledacq does extend symmetrically. If you wanted to improve the code here, you're welcome to steal that code ;-)